### PR TITLE
Pedestrian mode changes

### PR DESCRIPTION
**Pull request by na9da**
### What this PR does

* Limit up/down look angle #5194 
* Adds direction indicator to pedestrian mode minimap #5193 
* Automatically disable pedestrian mode when map zooms to a different location
* Styling changes #5318

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/Models/GlobeOrMap.ts [hunk 1]
```diff
@@ -39,17 +41,30 @@ export default abstract class GlobeOrMap {
...
   ): void;
+
+  zoomTo(
+    viewOrExtent: CameraView | Rectangle | Mappable,
+    flightDurationSeconds: number
+  ): void {
+    this.zoomToEvent.raiseEvent();
+    this.doZoomTo(viewOrExtent, flightDurationSeconds);
+  }
+
```
Comment (na9da) [inline lib/Models/GlobeOrMap.ts]: Raises an event when map zooms to a different location. Pedestrian mode will now disable itself when this event is fired.

Comment (KeyboardSounds) [reply lib/Models/GlobeOrMap.ts]: Where's the zoomToEvent handled by Pedestrian Mode? I can't find it 😬

File: CHANGES.md [hunk 2]
```diff
@@ -5,13 +5,16 @@ Change Log
...
 * Add CSW Group support
 * Revert "remove wmts interfaces from ows interfaces" (873aa70)
 * Add `math-expression-evaluator` library and `ColumnTransformationTraits`. This allows expressions to be used to transform column values (for example `x+10` to add 10 to all values).
 * Fix bug in `TableColumn.title` getter.
 * Add support for TableColumn quarterly dates in the format yyyy-Qx (eg 2020-Q1).
 * Fix region mapping feature highlighting.
 * Update clipboard to fix clipboard typing error.
+* Added direction indicator to the pedestrian mode minimap.
+* Limit up/down look angle in pedestrian mode.
 * [The next improvement]
```
Comment (KeyboardSounds) [inline CHANGES.md]: Please add:
```suggestion
* Automatically disable pedestrian mode when map zooms to a different location
* [The next improvement]
```

File: lib/Models/GlobeOrMap.ts (hunk 1 above)
Comment (na9da) [reply lib/Models/GlobeOrMap.ts]: Yeah it is easy to miss:

https://github.com/TerriaJS/terriajs/blob/f59b7b7b72ca4217faf9ad6725c900e9ed6d517b/lib/ReactViews/Tools/PedestrianMode/PedestrianMode.tsx#L30-L32

**Review by KeyboardSounds — APPROVED** [review]
Excellent job as always, Nanda!

**Pull request merged**